crates/flipperzero/src/furi/time.rs

```diff
@@ -0,0 +1,5 @@
+/// A placeholder `Instant` type. Currently unusable.
+pub struct Instant(pub(super) u32);
+
+/// A placeholder `Duration` type. Currently unusable.
+pub struct Duration(pub(super) u32);
```

Comment:
These being opaque with no operators effectively (temporarily) disables the "explicit timeout" `Mutex` APIs for now, while still keeping them implemented. Then once these are replaced by #74 the APIs will become usable.